include/flatbuffers/flatbuffers.h

```diff
@@ -393,6 +393,84 @@ template<typename T> static inline size_t VectorLength(const Vector<T> *v) {
   return v ? v->size() : 0;
 }
 
+// This is used as a helper type for accessing arrays.
+template<typename T, int length> class Array {
```

Comment:
The type of length should be matched with:
- uoffset_t size() const { return length; }
- short Type::fixed_length

Reply:
Changed to `uint16_t`.